Vagrantfile

```diff
@@ -232,6 +236,15 @@ Vagrant.configure("2") do |config|
           service NetworkManager restart
         SHELL
       end
+      # Hack for fedora39/40 to get the IP address of the second interface
+      if ["fedora39", "fedora40", "fedora39-arm64", "fedora40-arm64"].include? $os
+        config.vm.provision "shell", inline: <<-SHELL
+          nmcli conn modify 'Wired connection 2' ipv4.addresses $(cat /etc/sysconfig/network-scripts/ifcfg-eth1 | grep IPADDR | cut -d "=" -f2)/24
+          nmcli conn modify 'Wired connection 2' ipv4.method manual
+          service NetworkManager restart
+        SHELL
+      end
```

Comment:
I assume this can't be directly in the playbooks ? Why does fedora39/40 have two interfaces exactly ?

Reply:
Initially, I didn't configure this information, but etcd would fail to install, and the hosts couldn't connect to each other. (Fedora 37/38 has similar settings, too)